daprdocs/content/en/reference/components-reference/supported-bindings/mysql.md

```diff
@@ -55,21 +55,27 @@ Note that you can not use secret just for username/password. If you use secret,
 | `maxOpenConns` | N | Output | The max open connections. Integer greater than 0 | `"10"` |
 | `connMaxLifetime` | N | Output | The max connection lifetime. Duration string | `"12s"` |
 | `connMaxIdleTime` | N | Output | The max connection idel time. Duration string | `"12s"` |
-| `direction` | N | Output | The direction of the binding | `"output"` |
```

Comment:
Can we keep the `direction` on this component. We put these into all binding to be explicit that this is useful for separation of Dapr sidecar from asking the app.

Reply:
This is only an output binding so the direction is irrelevant. I think it causes more confusion than anything.

Reply:
It still tells the sidecar information that is needed. Even for output only bindings this is useful, yes?

Reply:
No, it is a no-op for output bindings, just adds confusion to users

Reply:
I am referring specifically to output-only bindings. A binding that is input-only or input and output can benefit from this. But output bindings don't need this